airmon_comm/AirSensData[] dataArray